.github/workflows/main.yml

```diff
@@ -63,6 +63,12 @@ jobs:
         run: cmake -DCMAKE_BUILD_TYPE=Release -DSUPERBUILD=ON -DBUILD_BACKEND=ON -j 2 -Bbuild/release -H.
```

Comment:
This will require the MAVLINK_PASSTHROUGH for building the examples

Reply:
Good point!